bindings/wasm/examples-account/src/revoke_vc.ts

```diff
@@ -86,7 +86,7 @@ async function revokeVC(storage?: Storage) {
 
     // Update the RevocationBitmap service in the issuer's DID Document.
     // This revokes the credential's unique index.
-    await issuer.revokeCredentials("my-revocation-service", 5);
+    await issuer.revokeIndices("my-revocation-service", 5);
```

Comment:
Not sure about the rename. I feel like "revoke indices" is not very clear in what it does. I think I still prefer `revokeCredentials` or perhaps `revokeCredentialIndices`?

Reply:
What about `revokeBitmapIndices`?

Reply:
Slightly better, just not entirely clear that it refers to indices that correspond to credentials, in my opinion.
I'm mainly concerned about readability. So when you read through an example and you see `account.revokeBitmapIndices("#my-revocation-service", 5);`, it doesn't say credential anywhere even though that's what it's ultimately about. But it's probably usually clear from context, so might be fine.

Reply:
Reverted to `revoke_credentials` for now until consensus can be reached. Should have been done in a different PR regardless.